include/libdnf5/repo/repo.hpp

```diff
@@ -327,6 +327,10 @@ class Repo {
     // @replaces libdnf:repo/Repo.hpp:method:Repo.getMirrors()
     std::vector<std::string> get_mirrors() const;
 
+    /// Returns baseurls of repository
+    /// @return A vector of baseurls
+    std::vector<std::string> get_baseurls() const { return config.get_baseurl_option().get_value(); };
```

Comment:
I am curios whether we need that. If I am not mistaken we already has several option like that, but do we want to make the same pattern for all configuration?

On the other hand it is not wrong and it might  be useful if the option is called often.

Reply:
It is not needed, I have dropped it.
We can always add it later if we want.